.vscode/settings.json

```diff
@@ -8,7 +8,6 @@
         ".github": false,
         "**/.pytest_cache": true,
         "**/__pycache__**": true,
-        "web/node_modules": true,
```

Comment:
just a personal preference. I'm always going in node modules. But other people want it hidden?

Reply:
I think it's fine. Where it can kind of get annoying is if you're doing a global string search in the repo and node modules are indexed those results can cloud what you're looking for.

Reply:
do you mean when you're searching using vscode? on my end at least if this button is clicked then it will ignore node_modules 
<img width="481" alt="image" src="https://github.com/user-attachments/assets/33a9f517-fdaf-4642-aeb6-fa90972121eb">

Reply:
Yeah in vscode, interesting ok, will check that button out.